gnucash/gnome-utils/gnc-main-window.cpp

```diff
@@ -3014,12 +3014,16 @@ gnc_main_window_engine_commit_error_callback( gpointer data,
  *
  *  @param menu_label The widget that should be added into the
  *  notebook popup menu for this page.  This should be a GtkLabel.
+ *
+ *  @param recreate The page is being recreated (while loading the book)
+ *  so always add the page the bottom and don't focus it after insertion.
```

Comment:
"add the page to the bottom" ?

And note that "the end" is probably better than "the bottom" because the default behavior is for the tabs to be displayed horizontally across the top.

Reply:
This code is no longer present.